pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/etterlatte/maler/omstillingsstoenad/informasjon/OmstillingsstoenadInformasjonDoedsfall.kt

```diff
@@ -0,0 +1,258 @@
+                        Nynorsk to "Vi oppmodar deg til å søkje så snart som mogleg fordi vi vanlegvis berre etterbetaler for tre månader. Det er nye reglar for ytingar til attlevande frå 1. januar 2024. Tidspunktet for når NAV mottar søknaden din kan ha betydning for kva regler som gjeld for deg.  ",
+                        English to "If the deceased was a member of a private or public pension scheme and you have questions about this, you can contact the deceased's employer. You can also contact the pension scheme or insurance company.  ",
+                    )
+                }
+                paragraph {
+                    text(
+                        Bokmal to "Dersom du bor i utlandet, må du kontakte trygdemyndigheten i bostedslandet ditt. ",
+                        Nynorsk to "Dersom du bur i utlandet, må du kontakte trygdemyndigheitene i bustadlandet ditt. ",
+                        English to "If you are receiving tariff-based early retirement pensions (AFP) in the public sector, you should contact us for further guidance. ",
+                    )
+                }
+
+
+
+                title2 {
+                    text(
+                        Bokmal to "Rettigheter hvis avdøde har bodd eller arbeidet i utlandet",
+                        Nynorsk to "Rettar når avdøde har budd eller arbeidd i utlandet",
+                        English to "Rights if the deceased has lived or worked abroad",
+                    )
+                }
+                paragraph {
+                    text(
+                        Bokmal to "Hvis avdøde har bodd eller arbeidet i utlandet, kan dette få betydning for hvor mye du får ubetalt. Norge har trygdesamarbeid med en rekke land gjennom EØS-avtalen og andre avtaler. Derfor kan du også ha rettigheter fra andre land. Vi kan hjelpe deg med søknad til land Norge har trygdeavtale med.",
+                        Nynorsk to "Dersom avdøde har budd eller arbeidd i utlandet, kan det påverke kor mykje du får ubetalt. Noreg har trygdesamarbeid med ei rekkje land gjennom EØS-avtalen og andre avtalar. Derfor kan du også ha rettar frå andre land. Vi kan hjelpe deg med søknad til land Noreg har trygdeavtale med.",
+                        English to "If the deceased has lived or worked abroad, this may affect the amount of your pension. Norway cooperates with a number of countries through the EEA Agreement and other social security agreements. Therefore, you may also be entitled to a pension from other countries. We can assist you with your application to countries with which Norway has a social security agreement.",
+                    )
+                }
+
+                title2 {
+                    text(
+                        Bokmal to "Andre ytelser du kan ha rett til som gjenlevende ",
+                        Nynorsk to "Andre ytingar du kan ha rett til som attlevande ",
+                        English to "",
```

Comment:
Er det meningen at det ikke skal være engelsk oversettelse for denne tittelen?

Reply:
Jepp det er et helt avsnitt+tittel som ikke finnes i engelsk malen